http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/WelderConfig.scala

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+final case class WelderConfig(
+  welderEnabledNotebooksDir: String,
```

Comment:
does it make sense to use `java.nio.file.Path`?

Reply:
👍